tests/models/test_viscoelasticity.py

```diff
@@ -291,7 +291,7 @@ def test_kelvin_vs_maxwell():
     """1D test with uniaxial stress, compare Kelvin and Maxwell model."""
 
     mesh = df.mesh.create_unit_interval(MPI.COMM_WORLD, 2)
-    V = df.fem.FunctionSpace(mesh, ("CG", 1))
+    V = df.fem.functionspace(mesh, ("CG", 1))
```

Comment:
Sometimes you have ("CG",1 (dim,)) Here the dim part is missing. Why? Is it default 1?